static/script/historian.js

```diff
@@ -121,16 +122,16 @@ define(
              * @returns {Boolean} True if the history stack contains one or more valid return URLs.
              */
             hasHistory: function hasHistory () {
-                var historyMinimumLength = this.hasBroadcastOrigin() ? 2 : 1;
+                var historyMinimumLength = 1;
                 return this._historyArray.length >= historyMinimumLength;
             },
 
             /**
-             * Returns a Boolean to indicate whether the first entry in the history stack is the special 'broadcast' entry.
-             * @returns {Boolean} True if the first entry in the history stack is the special 'broadcast' entry.
+             * Returns a Boolean to indicate whether the URL contains a 'broadcast' query parameter.
+             * @returns {Boolean} True if the URL contains a 'broadcast' query parameter.
```

Comment:
We should mark this as `@deprecated` so that hasBroadcastOrigin() can be removed from Historian in the next major release, now that broadcast origin is indicated elsewhere in the URL.